src/routes/middleware/validators/recipe.validator.ts

```diff
@@ -0,0 +1,21 @@
+import { body } from "express-validator";
+
+const title = body("title").notEmpty();
+const description = body("description").notEmpty();
+const url = body("url").notEmpty();
+const id = body("id").notEmpty();
+const preparing = body("preparing").isArray();
+const ingredients = body(["ingredients", "ingredients[*]"]).isArray();
+
+const validateData = [title, description, preparing, ingredients, url];
+
+export class RecipeValidator {
```

Comment:
Not sure if class is required here (could just create and export consts) but ok.

Reply:
or perhaps it's better this way? I'm not sure, leave it as it is